assets/chat/css/style.scss

```diff
@@ -1317,6 +1317,16 @@ hr {
     background: none;
     border-radius: 0;
   }
+  .all {
+    flex: 1;
+    height: auto;
+  }
+  .favorite {
+    flex: 0;
+    height: auto;
+    min-height: 88px;
```

Comment:
Is the goal here to make the favorite panel a fixed 88px and have the all panel take up the remaining space? If so, you can simply give the favorite panel
```
flex: 0 88px;
```
which sets its initial height to 88px and tells it not to grow. With this rule alone, I believe both panels will behave as intended. You can remove everything else.